test/node/rest-api/request/body/body-protobuf.node.test.ts

```diff
@@ -0,0 +1,38 @@
+// @vitest-environment node
+import { http, HttpResponse } from 'msw'
+import { setupServer } from 'msw/node'
+
+const server = setupServer(
+  http.post('https://example.com/protobuf', async ({ request }) => {
+    const buffer = await request.arrayBuffer()
+
+    return new HttpResponse(new Uint8Array(buffer), {
```

Comment:
@DGollings, hi 👋 Do you think it's necessary to preserve this conversion here or should be simply forward the `buffer` as-is to the response? It'd also change how we assert the actual response body sent back. 

In any case, the test is now passing.

Reply:
I honestly don't remember why that conversion is there, maybe to be slightly more realistic to actual usage? You generally don't respond with the exact request as-is (I think? Its been over a year)